downstream.stages

```diff
@@ -1,11 +1,14 @@
 stage('Downstream Build') {
     final REPOSITORY_LIST_FILE = "./script/repository-list.txt"
     final SETTINGS_XML_ID = "771ff52a-a8b4-40e6-9b22-d54c7314aa1e"
+    final PROPERTIES_FILE_ID = "387b2726-55c0-4fcf-b5c0-ee7f4174d58a"
     configFileProvider([configFile(fileId: SETTINGS_XML_ID, variable: 'MAVEN_SETTINGS_XML_DOWNSTREAM')]) {
         println "Reading file ${REPOSITORY_LIST_FILE}"
         def file = readFile REPOSITORY_LIST_FILE
         def projectCollection = file.readLines()
-        projectCollection.removeAll(['kie-jpmml-integration','droolsjbpm-tools'])
-        treebuild.build(projectCollection, "${SETTINGS_XML_ID}", '-e -nsu -Pbusiness-central,wildfly,sourcemaps,no-showcase clean install -Dfull=true -Dcontainer=wildfly -Dcontainer.profile=wildfly -Dintegration-tests=true -Dmaven.test.failure.ignore=true -Dmaven.test.redirectTestOutputToFile=true -Dgwt.compiler.localWorkers=1 -Dwebdriver.firefox.bin=/opt/tools/firefox-60esr/firefox-bin')
+        def gitURL = env.ghprbAuthorRepoGitUrl ?: env.GIT_URL
+        def project = util.getProjectGroupName(util.getProject(gitURL))[1]
+        projectCollection.removeAll(['droolsjbpm-tools', 'kie-docs'])
+        compile.build(projectCollection, project, "${SETTINGS_XML_ID}", "${PROPERTIES_FILE_ID}")
```

Comment:
@mbiarnes why do you call `compile` from the downstream build?

Reply:
are we using compile for the new downstreams? :thinking:

Reply:
@ginxo because we had this from the beginning. All test end everything were done with this. 
We only have in shared libraries or pullrequest.groovy o compile.groovy, there is not (yet) existing a downstream.groovy.
Should it be treebuild.groovy?